kibana.yml

```diff
@@ -32,6 +32,8 @@
 # is proxied through the Kibana server.
 # elasticsearch.username: "user"
 # elasticsearch.password: "pass"
+%ELASTICSEARCH_USERNAME%
```

Comment:
This is a bit ugly I agree but with default empty values for both username and password this seem to break configuration. The workaround for that it to include entire line only if username / password is specified.